frontend/react/.nvmrc

```
@@ -0,0 +1 @@
+14.0.0
```

```suggestion
14.4.0
```

I think to match the [Dockerfile](https://github.com/CMSgov/cms-carts-seds/blob/424221167fbc5f08103ce4bc0fe4175ace91e83f/frontend/react/Dockerfile#L3) and the [Jenkinsfile](https://github.com/CMSgov/cms-carts-seds/blob/424221167fbc5f08103ce4bc0fe4175ace91e83f/.jenkins/Jenkinsfile.master#L39).

---

Done! Thanks for the heads up.